tests/savings.validation.test.ts

```
@@ -0,0 +1,105 @@
+  });
+
+  it("rejects savings positions when term_seconds is invalid", () => {
+    const req = {
+      body: {},
+      query: { term_seconds: "not-a-number" },
+      params: {},
+    } as unknown as Request;
+
+    expect(() => validate(savingsPositionsSchema)(req, mockRes, mockNext)).toThrow(
+      AppError,
+    );
```

ESLint

[error] Replace `·validate(savingsPositionsSchema)(req,·mockRes,·mockNext)).toThrow(⏎······AppError,⏎····` with `⏎······validate(savingsPositionsSchema)(req,·mockRes,·mockNext),⏎····).toThrow(AppError`

(prettier/prettier)